examples/anim.rs

```diff
@@ -136,7 +136,7 @@ fn main() {
     let blend_time = 0.2f32;
     let mut timer = win.input.time();
 
-    while win.update() && !three::KEY_ESCAPE.is_hit(&win.input) {
+    while win.update() && !win.input.button_down(three::KEY_ESCAPE) {
```

Comment:
careful with semantics! What this should be doing is checking if `Esc` was hit *at some point* between the last frame and this one. This is not the same as checking for the `Esc` to be *currently* pressed, which is what `button_down` implies to me.

Reply:
Then `is_hit` is a bad naming too. Perhaps we can use `button_pressed`?